config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: myflix_development
+  pool: 5
+  username: boottest
+  password: Iputo*34

```

Comment:
Not a big deal now, but you'll learn how to handle such sensitive info later in the course